chall04/rjaakonm.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+import sys
+
+def main():
+	lines = []
+	counted = 0
+	for line in sys.stdin:
+		lines.append(list(line))
+		counted += 1
```

Comment:
This could be written in a more pythonic way.

```python
grid = [list(input()) for _ in range(n)]
```
and I think `counted` could be use another way: `len(grid)`